infra/helper.py

```diff
@@ -1703,13 +1712,28 @@ def index(args):
 
   run_args = _env_to_docker_args(env)
   run_args.extend([
-      '-v', f'{args.project.out}:/out', '-v', f'{args.project.work}:/work',
-      '-t', image_name, '/opt/indexer/index_build.py'
+      '-v',
```

Comment:
While you are at it, I would recommend writing this out in the `--volume=/foo:/bar` style as well, for understandability (flag meaning is clearer) and so that it fits on one line each.

Reply:
Hmm, I'd prefer to leave this as is for now, for consistency with the rest of this file, where we are using short flags.